lib/iris/tests/unit/util/test_file_is_newer_than.py

```diff
@@ -44,7 +42,7 @@ def setUp(self):
             mtime += 5.0 + 10.0 * i_file
             os.utime(file_path, (mtime, mtime))
 
-    def tearDown(self):
+    def teardown_method(self):
```

Comment:
You can use the `setup -> yield -> teardown` style all within one fixture.

But in this case there is a built-in mechanism where you won't need a teardown at all:

[How to use temporary directories and files in tests — pytest documentation](https://docs.pytest.org/en/stable/how-to/tmp_path.html)